src/osx/webrequest_urlsession.mm

```diff
@@ -434,6 +434,12 @@ void Wait()
         return wxString();
 }
 
+std::vector<wxString> wxWebResponseURLSession::GetAllHeaderValues(const wxString& WXUNUSED(name)) const
+{
+    std::vector<wxString> result;
+    return result;
```

Comment:
It's fine not to implement this for macOS if you don't know how to do it but I'd at least call `GetHeader()` here as returning the first header value seems like a more useful fallback than not returning anything at all.

This behaviour should be mentioned in the documentation, of course.